### PR TITLE
First Pull request

### PR DESCRIPTION
**Pull request by minh711**
Request merge from a fork.

### CODE REVIEW

**Comment by minh711-2** [comment]
Thanks for requesting, I will check it later. :)

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# test-pull-request
\ No newline at end of file
+# test-pull-request
+
+# First pull request
+
+Directly from `main`, from another account.
```
Comment (minh711-2) [inline README.md:5]: Ok

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+Using **VSCode** UI.
+
+Need a **fork**.
+
+Now it has a option:
+
+![Pull request option](images/image.png)
```
Comment (minh711-2) [inline README.md:13]: Yes

File: README.md (hunk 1 above)
Comment (minh711) [reply README.md:5]: Thanks :)

File: README.md (hunk 2 above)
Comment (minh711) [reply README.md:13]: Yes

**Review by minh711-2 — APPROVED** [review]
All's good!

**Pull request merged**